src/aleph/vm/resources.py

```diff
@@ -71,6 +72,8 @@ def get_gpu_model(device_id: str) -> bool | None:
 def is_gpu_compatible(device_id: str) -> bool:
     """Checks if a GPU is compatible based on vendor and model IDs."""
     compatible_gpu_set = {gpu["device_id"] for gpu in get_compatible_gpus()}
+    logger = logging.getLogger(__name__)
+    logger.info(get_compatible_gpus())
```

Comment:
```suggestion
    logger = logging.getLogger(__name__)
    logger.debug(get_compatible_gpus())
```
I think this log output can be moved to debug instead info

Reply:
you are right, I had put that there because I had an issue with my debuger setup at the time. I remove the line completely